src/OhmmsSoA/VectorSoaContainer.h

```diff
@@ -47,7 +47,7 @@ struct VectorSoaContainer
   ///allocator
   Alloc myAlloc;
   ///default constructor
-  VectorSoaContainer() { setDefaults(); }
+  VectorSoaContainer() : nLocal(0), nGhosts(0), nAllocated(0), myData(nullptr) {}
```

Comment:
I think we should fully replace all setDefaults. is there any left?

Reply:
I'll check.